clang/test/ClangScanDeps/cas-trees.c

```diff
@@ -0,0 +1,11 @@
+// RUN: rm -rf %t && mkdir %t
+// RUN: cp -r %S/Inputs/textual/* %t
+// RUN: sed -e "s|DIR|%/t|g" %t/cdb.json.template > %t/cdb.json
```

Comment:
The "cool kids" are doing testcases like this that have multiple files using split-file these days. Much easier to reason about later since it's easy to see them all at once:
https://www.llvm.org/docs/TestingGuide.html#extra-files

Up to you whether to convert this.

Reply:
Oh yes, this is much better! It also helps with `CHECK`ing line numbers that come from these files, which is useful for something else I'm working towards.